roles/slurm-client/handlers/main.yml

```diff
@@ -10,14 +10,16 @@
 - name: Restart munge service.
   systemd:
     name: 'munge.service'
-    state: restarted
+    state: 'restarted'
+    daemon_reload: 'yes'
```

Comment:
I'm not sure if i'm a fan of quoting everything that can be quoted. But the linter doesn't care either way so i'm not going to force my taste on you.